umh-core/integration/docker_test_helpers.go

```
@@ -0,0 +1,93 @@
+)
+
+const (
+	containerName = "umh-core"
+	imageName     = "umh-core:latest"
+	metricsURL    = "http://localhost:8081/metrics"
+)
+
+// BuildAndRunContainer rebuilds your Docker image, starts the container, etc.
+func BuildAndRunContainer(configFilePath string, memory string) error {
+	// 1. Stop/Remove the old container if any
+	runDockerCommand("rm", "-f", containerName) // ignoring error
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)